modules/stackexchange/stackexchange.go

```diff
@@ -0,0 +1,107 @@
+package stackexchange
+
+import (
+	"github.com/PuffyVulva/Stack-on-Go/stackongo"
+	"github.com/lovelaced/glitzz/config"
+	"github.com/lovelaced/glitzz/core"
+
+	"errors"
+	"html"
+	"math/rand"
+	"strings"
+)
+
+var (
+	// Put your default tags here
+	defaultTags []string
+	lastLink    string
+)
+
+const (
+	errInvalidSite = "Invalid site: "
+	errInvalidTags = "Tag(s) not found"
+	defaultSite    = "workplace"
+	sePrefix       = "se"
+	seLast         = "last"
+)
+
+// se = seprefix
+// seprefix: seq (question title)
+// seq: question title
+
+// New initializes the stackexchange module
+func New(sender core.Sender, conf config.Config) (core.Module, error) {
+	rv := &stackexchange{
+		Base: core.NewBase("stackexchange", sender, conf),
+	}
+	rv.AddCommand("so", rv.seStackOverflow)
+	rv.AddCommand(sePrefix, rv.seTitle)
+	rv.AddCommand(sePrefix+seLast, rv.seLastLink)
+	return rv, nil
+}
+
+type stackexchange struct {
+	core.Base
+}
+
+func (s *stackexchange) getSite(arguments core.CommandArguments) string {
+	if len(arguments.Arguments) > 0 {
+		return arguments.Arguments[0]
+	}
+	return defaultSite
+}
+
+func (s *stackexchange) getTags(arguments core.CommandArguments) []string {
+	if len(arguments.Arguments) > 1 {
+		return arguments.Arguments[1:]
+	}
+	return defaultTags
+}
+
+func (s *stackexchange) getRandQuestion(arguments core.CommandArguments) (*stackongo.Question, error) {
+	site := s.getSite(arguments)
+	tags := s.getTags(arguments)
+
+	session := stackongo.NewSession(site)
```

Comment:
This library is broken. It has global behavior (setTransport) and no way to set the http client. 

https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779